meta-refkit-extra/recipes-multimedia/btspeaker/btspeaker.bb

```diff
@@ -0,0 +1,90 @@
+SUMMARY = "Bluetooth speaker"
+DESCRIPTION = "This recipe configures refkit gateway image into a stand alone bluetooth speaker"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${META_REFKIT_BASE}/COPYING.MIT;md5=838c366f69b72c5df05c96dff79b35f2"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRC_URI = "\
+        file://btspeaker.py \
+        file://btspeaker.service \
+        file://client.conf \
+        file://leds.rules \
+        file://btspeaker-bluetooth.conf \
+        file://btspeaker-connman.conf \
+        file://pulseaudio-bluetooth.conf \
+        file://audio.conf \
+        file://buttons.asl \
+        file://leds.asl \
+        file://btpairing.wav \
+        file://btstartup.wav \
+        file://btsuccess.wav \
+"
+
+DEPENDS = "acpica-native"
+
+RDEPENDS_${PN} = "\
+        python-dbus \
+        python-pygobject \
+        python-pyalsaaudio\
+        python-evdev \
+"
+
+inherit useradd deploy systemd
+
+USERADD_PACKAGES = "${PN}"
+GROUPADD_PARAM_${PN} = "-g 1100 -r leds"
+USERADD_PARAM_${PN} = "-u 1200 -d /home/btspeaker -r -s /bin/bash -G audio,leds,input btspeaker"
```

Comment:
do you mean specifically bash or shell in general? This is pretty good for debugging....